partner.md

```diff
@@ -1,47 +1,103 @@
-6. Do you use desktop programs to record information so you are locked into doing it from a certain location but it would be more convenient to have it web based so you can see/edit from anywhere?
-
-We can and have built other kinds of software and this is only to give you an idea. We'll definitely find a way to help!
-
-Listen to what one of our amazing partner organizations from 2017 had to say:
-
-<iframe width="560" height="315" src="https://www.youtube.com/embed/kON0NSr6qk8" frameborder="0" allowfullscreen></iframe>
-
-
-#### What do we need from you?
-
-**Involvement and Participation**
-
-In order to build you the best software possible we need your participation so we can understand your needs. We require you to commit to at least two meetings with the person leading your team in the lead up to the event.
-
-You will also need to be available at least once during the event, ideally each day. Your team will coordinate the times. This is necessary because the team building your software will inevitably run into design decisions that will require your input in order to resolve and keep the project moving forward.
-
-#### What does it cost?
-
-The project development is free! We do this to help the people who are making the world a better place. For the past three years we have also been very fortunate to have received sponsorships from hosting companies to host the projects. However, there is no guarantee these sponsorships will continue. You may need to find hosting for the project once it is built. We will work with you to find the most affordable solution for your project or to integrate it with your current hosting.
-
-#### Okay, I'm sold, how do I get involved?
-
-The process starts with a conversation. We'll get in touch and determine how we can best help you. It may turn out that your best solution is to use something already out there and we'll help you with that.
-- About 5-6 weeks before the event we'll find someone suitable to lead a team to build your software and put them in touch with you.
-They will meet and drill down with you and what you really need. They will confirm requirements with you and then present them to the team that will be working on your software at our event.
-They will meet virtually (Skype, Google Hangouts, etc.) with you during the event to introduce the team and to clarify any questions.
-Within one month of the event's completion we will deliver the software to you.
-
-[Email us](mailto:{{ site.data.constants.contact_email }}) or submit your project [here](https://docs.google.com/forms/d/e/1FAIpQLScfGdu_Tx5UFoQ8zC2LgMCUj46RTRmx0Sfj1I1VuTYmvzo1Zw/viewform).
+<div class="col-md-4 col-sm-4 hidden-xs" markdown="1">
+  <img class="img-responsive" src="/assets/img/otter.png" alt="ruby-by-the-bay" />
```

Comment:
The otter's background color is different from the webpage's background color - maybe use an otter with a transparent background?

Reply:
perfect! i didn't have that file but I agree that would be much better